src/components/signin/components/Signin.js

```diff
@@ -40,25 +40,29 @@ class Signin extends React.Component {
             signinRequest(payload);
             this.setState({ visible: false });
         });
-        
+
     }
 
     saveFormRef = formRef => {
         this.formRef = formRef;
     }
     notifySigninStatus = () => {
+
+    }
+
+    componentDidMount() {
```

Comment:
I see what you did here

Reply:
What is going on here

Reply:
I moved the code in a react life cycle, I will go over them briefly during our meeting today